deeplake/core/dataset/dataset.py

```diff
@@ -1633,6 +1633,13 @@ def query(self, query_string: str):
 
             >>> ds_train = deeplake.load('hub://activeloop/coco-train')
             >>> query_ds_train = ds_train.query("(select * where contains(categories, 'car') limit 1000) union (select * where contains(categories, 'motorcycle') limit 1000)")
+
+        **Restrictions**
+
+        Querying datasets is part of our Growth and Enterprise Plan .
+
+        - Users of our Community plan can only perform queries on Activeloop datasets ("hub://activeloop/..." datasets).
+        - To run queries on your own datasets, `upgrade your organization's plan <https://app.activeloop.ai/pricing>`_.
```

Comment:
This link won't quite work. Pls use the one below for pricing.

https://www.activeloop.ai/pricing/